src/test.ts

```diff
@@ -1,13 +1,43 @@
 import test from 'ava'
 import {generateNumber, normalizeNumber} from '.';
 
-test(async t => {
-	t.is(generateNumber().length, 6);
-	t.is(generateNumber(10).length, 10);
-	t.is(generateNumber(10, 45).length, 10);
+function isSortedList(list: number[]) {
+  const recursiveSortCheck = (head: number, rest: number[]) => {
+    if (rest.length === 0) return true;
+    if (head > rest[0]) return false;
+
+    const [nextHead, ...nextRest] = rest;
+    return recursiveSortCheck(nextHead, nextRest);
+  }
+
+  const [head, ...rest] = list;
+  return recursiveSortCheck(head, rest);
+}
+
+test('isSortedList', async t => {
+  const sortedList = [1, 2, 3, 4, 5, 6]
+  const notSortedList = [7, 10, 2, 58, 1, 40]
+
+  t.is(isSortedList(sortedList), true);
+  t.is(isSortedList(notSortedList), false);
+})
+
+
+test('generateNumber', async t => {
+  const defaultLength = generateNumber();
+  const longerNumbers = generateNumber(10);
+  const lonegerNumbersWithLimit = generateNumber(10, 45);
+
+  t.is(isSortedList(defaultLength), true);
```

Comment:
dup, we already have it above. checking length only

Reply:
But if I test only length of `generateNumber()`, how can I be sure the generated numbers are sorted in the test case?

Reply:
~~Sorting test is added from your PR~~. (I think sorting test is added at first) testing should be isolated and separated. You should add a new test for sorting not updated length test even it was not tagged.

test('sorting' ...)
test('length' ...)
test('normalize' ...)

Reply:
ahh, I misunderstood what you mentioned. I'll try it again.